examples/react18/src/App.tsx

```diff
@@ -15,8 +15,13 @@ const handleFailRequestsOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
   state.failRequests = e.target.checked;
 };
 
+if (!process.env.REACT_APP_CLIENT_SECRET) {
+  console.error('REACT_APP_CLIENT_SECRET not set in .env');
```

Comment:
Are we considering something similar for the other example apps?

Reply:
I can add it!